advanced-topics/network-configuration.md

```diff
@@ -0,0 +1,145 @@
+# Network Configuration
+
+In your infrastructure, Plural will create a VPC, a public ingress controller, and a private ingress controller. Sometimes your organization or project will want to change our networking defaults to 
+better suit your deployment.
+
+## VPC Subnet configuration
+
+You’ll want to overwrite our default subnet configuration if you need to add the VPC that Plural creates to any adjacent VPCs/networks in your infrastructure.  
+
+Frequently, VPC peering APIs require all subnets to be non-overlapping, which isn't guaranteed from our defaults.  This can always be updated in `bootstrap/terrraform/main.tf` (As a reference, CIDR blocks follow this format: `10.xx.xx.xx/yy`, the examples below are placeholders.)
+
+{% tabs %}
+{% tab title="AWS" %}
+On AWS, update the `aws-bootstrap` modules configuration with:
+
+```bash
+public_subnets = ["your.cidr.pub.1", "your.cidr.pub.2", "your.cidr.pub.3"]
+private_subnets = ["your.cidr.priv.1", "your.cidr.priv.2", "your.cidr.priv.3"]
+worker_private_subnets = ["your.cidr.worker.1", "your.cidr.worker.2", "your.cidr.worker.3"]
+```
+{% endtab %}
+
+{% tab title="GCP" %}
+On GCP, update the `gcp-bootstrap` modules configuration with:
+
+```bash
+vpc_subnetwork_cidr_range = "your.cidr"
+
+# you might also want to update cluster_secondary_range_cidr 
+# and services_secondary_range_cidr
+```
+
+{% endtab %}
+
+{% tab title="Azure" %}
+With Azure, update the `azure-bootstrap` modules configuration with:
+
+```bash
+address_space = "your.cidr"
+subnet_prefixes = ["your.cidr.pref"]
+```
+
+{% endtab %}
+{% endtabs %}
+
+{% hint style="warning" %}
+💡 Note that updating these will likely cause the VPC to be replaced, which will recreate your cluster. We recommend that you destroy your cluster before applying network config modifications to it.
+{% endhint %}
+
+## Configuring VPC Peering
+
+Plural creates a fresh VPC and Kubernetes cluster for deploying your applications into. This ensures that we have a clean environment to deploy into and minimizes disruption to existing systems. It does come with the tradeoff of slightly increased network complexity, but most cloud providers can bridge this using VPC peering. This is a technology that effectively allows you to combine the address spaces of two VPCs in the cloud you operate in.
```

Comment:
Probably should rm the trailing into on the first sentence, or reword it somehow.

Reply:
Yep, makes sense - fixed.